integrations/jenkins/Jenkinsfile

```diff
@@ -673,8 +673,7 @@ pipeline {
                 fi
 
                 /opt/ort/bin/ort $ORT_OPTIONS report \
-                    -f CycloneDX,PlainTextTemplate,SpdxDocument,StaticHTML,WebApp \
-                    -O PlainTextTemplate=template.id=NOTICE_DEFAULT,NOTICE_SUMMARY \
+                    -f CycloneDX,SpdxDocument,PdfTemplate,PlainTextTemplate,StaticHTML,WebApp \
```

Comment:
Previously reporter names were in alphabetical order. Should we sort the entries again?

Reply:
I felt like roughly grouping them by type (SBOMs, attributions, reports) instead. But I can revert to strictly alphabetical sorting if you want.